src/fzf.c

```diff
@@ -1048,7 +1048,7 @@ static fzf_result_t fzf_call_alg(fzf_term_t *term, bool normalize,
  * - always v2 alg
  * - bool extended always true (thats the whole point of this isn't it)
  */
-fzf_pattern_t *fzf_parse_pattern(fzf_case_types case_mode, bool normalize,
+DLLEXPORT fzf_pattern_t *fzf_parse_pattern(fzf_case_types case_mode, bool normalize,
```

Comment:
You can get rid of this and use [WINDOWS_EXPORT_ALL_SYMBOLS](https://cmake.org/cmake/help/latest/prop_tgt/WINDOWS_EXPORT_ALL_SYMBOLS.html) in CMake.